### PR TITLE
types and tests for prepareOwnable

### PR DESCRIPTION
**Pull request by turadg**
## Description

Give is a return type (was `any`) and adds a test for attenuation behavior

### Other Considerations
doesn't impact production

### CODE REVIEW

File: packages/zoe/test/types.test-d.ts [hunk 1]
```diff
@@ -20,13 +20,13 @@ import type { prepare as scaledPriceAuthorityStart } from '../src/contracts/scal
   const kit = await E(zoe).startInstance(scaledPriceInstallation);
   // @ts-expect-error
   kit.notInKit;
-  E(kit.publicFacet).getPriceAuthority();
+  void E(kit.publicFacet).getPriceAuthority();
```
Comment (erights) [inline packages/zoe/test/types.test-d.ts:23]: There are driveby fixes, yes? Meaning, they aren't related to the purpose of this PR but a fix made in passing and bundled in. I'm all in favor; just making sure I understand.

File: packages/zoe/test/unitTests/contracts/ownable-counter.js [hunk 2]
```diff
@@ -44,6 +48,7 @@ export const start = async (zcf, privateArgs, baggage) => {
             count,
           });
         },
+        toBeAttenuated() {},
```
Comment (erights) [inline packages/zoe/test/unitTests/contracts/ownable-counter.js:51]: Nice additional test.

**Review by erights — APPROVED** [review]
LGTM, thanks for doing this!

File: packages/zoe/test/types.test-d.ts (hunk 1 above)
Comment (turadg) [reply packages/zoe/test/types.test-d.ts:23]: Yep, separate commit [6dfc8e3](https://github.com/Agoric/agoric-sdk/pull/9092/commits/6dfc8e326604bc8325b4205453eb868a81825168)
